terraform/environments/electronic-monitoring-data/modules/landing_bucket/outputs.tf

```
@@ -1,3 +1,7 @@
 output "bucket_id" {
   value = module.this-bucket.bucket.id
+}
+
+output "bucket_arn" {
```

im guessing youve tried the below and it doesnt work?
```suggestion
output "bucket" {
  value = module.this-bucket.bucket
```

---

im not sure the suggestion will acc work if you commit it tho lol

---

not sure what you're angling for here anyway!